### PR TITLE
Math Magicians: Components

### PR DESCRIPTION
**Pull request by raihan2bd**
# Math Magicians 🚀 

Hi

**Hi There 👋**  In this PR
- [x] I created new branch then and deleted unwanted resources from my project.✔️ 
- [x] Then Added the Calculator component in my project components folder.✔️
- [x] Modified the readme file to make it professional.✔️

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -9,7 +9,7 @@
 # 📗 Table of Contents
 
 - [📗 Table of Contents](#-table-of-contents)
-- [🎥 RSL Media ](#-rsl-media-)
+- [ 💻 Math Magicians ](#-math-magicians-)
```
Comment (TedLivist) [inline README.md:12]: - [x] Kindly remove the following highlighted boilerplate files as they would not be needed.

  image

File: src/components/Calculator.js [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+              AC
+            </button>
+            <button className="btn-default" type="button">
+              +/-
+            </button>
+            <button className="btn-default" type="button">
+              %
+            </button>
+            <button className="btn-default btn-orange" type="button">
+              %
```
Comment (TedLivist) [inline src/components/Calculator.js]: - [x] You duplicated the "%" sign, but you should have this instead "&div;". Kindly fix that.

  image

**Review by TedLivist — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED ♻️ ♻️

Hi @raihan2bd,

Good job so far!
✔️ Good documentation
✔️ Use of class components
However, there are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. And please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/Calculator.js (hunk 2 above)
Comment (raihan2bd) [reply src/components/Calculator.js]: Hi @TedLivist 👋,   Thanks a lot for valuable suggestions. 🙏  I will appy your suggestions in my project.  🙂

**Review by cynthiainga — APPROVED** [review]
Hi @raihan2bd, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights

- [x] No linters errors ✔️ 
- [x] Descriptive PR ✔️ 
- [x] Correct Gitflow ✔️ 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**